features/org.wso2.carbon.appmgt.store.feature/src/main/resources/store/extensions/assets/site/pages/assets.jag

```diff
@@ -207,16 +207,53 @@ require('/modules/store.js').exec(function (ctx) {
         options.lifecycleState = ["published"];
         assets = store.taggeds(type, options, paging);
     } else if (query) {
-        var urlQuery = 'query=' + query + '&';
-        query = require('/modules/search.js').build(query);
-        query.overview_treatAsASite = 'TRUE';
-        if (!isMultipleVersionDisplayEnabled) {
-            query.overview_makeAsDefaultVersion = 'true';
+        var searchParameter;
+        var searchValue;
+        var splitQuery = query.split(":");
+        searchParameter = splitQuery[0];
+        // This is a registry based search. In search box, user wants to search by business owner name. But
+        // in registry, there is only business owner Id. Therefore, it is necessary to get business owner Ids
+        // related to search prefix from DB.
+
+        if (!searchParameter.equals("Business Owner")) {
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/4642888b8ad6934dcda162142607b8549b0728e5